filters/include/pcl/filters/functor_filter.h

```diff
@@ -76,22 +77,40 @@ class FunctorFilter : public FilterIndices<PointT> {
    */
   void
   applyFilter(Indices& indices) override
+  {
+    applyFilter(executor::best_fit{}, indices);
+  }
+
+  template <typename Executor, typename std::enable_if_t<executor::is_instance_of_base_v<executor::inline_executor, Executor> || executor::is_instance_of_base_v<executor::omp_executor, Executor>, int> = 0>
+  void
+  applyFilter (Executor &&exec, Indices& indices)
   {
     indices.clear();
-    indices.reserve(input_->size());
+    indices.reserve(indices_->size());
+    std::vector<int> keep(indices_->size(), 0);
+    std::vector<int> remove(indices_->size(), 0);
```

Comment:
Is there a point in having this vector given that `keep` is looking very much like the negated version of `remove`.